template-html5-app/src/index.ts

```diff
@@ -0,0 +1,27 @@
+import { displayCurrentNetwork, displayDeviceInfo } from "./info";
+import { getConfig } from "./config";
+
+const config = getConfig();
+
+const start = async () => {
+    console.log(
+        `config.env: desktop mode - ${config.isDesktop}, environment - ${config.nodeEnv}`
+    );
+
+    if (!config.isDesktop) {
+        displayCurrentNetwork();
+        await displayDeviceInfo();
+    }
```

Comment:
@mingjun7 this line is where the web socket would open in the example you've been developing. I removed this as web socket is not relevant for this template. Should I be starting anything else up here instead?

Reply:
It's really your choice what functionality you want to support when running on desktop. If the html can be running in desktop mode, then you can apply this part of code as well.

I did not trigger the code on desktop because the html info is not really necessary to me, and I'll actually switch to use a node app in the future.

Reply:
The template now opens a browser and hosts the index.html file. Thanks for the info :)